mod/update_notes.php

```diff
@@ -16,7 +16,7 @@
 
 function update_notes_content(App $a)
 {
-	$profile_uid = intval($_GET['p']);
+	$profile_uid = intval($_GET['p'] ?? 0);
```

Comment:
When we can't fetch the user id, I suggest to quit here. That's much safer. We shouldn't proceed here with an `uid' of `0`.

Reply:
More context: for technical reasons, we do have a node-wide system contact/profile with a `uid = 0`.